starter-workflows/README.md

```diff
@@ -0,0 +1,12 @@
+## Starter Workflows
```

Comment:
## Starter Workflows
-->
## GitHub Starter Workflows for Frogbot

Reply:
done